src/core/issues-notes.js

```diff
@@ -23,6 +23,16 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    if (!(Object.keys(ghIssues).length === 0 && ghIssues.constructor === Object)) {
```

Comment:
Line 140 (150 in your PR) always creates `ghIssues` as an object, so probably no need to check constructor. Probably worth making `ghIssues` as a map rather than a dictionary object, so that we can just check `ghIssues.size` instead of doing `Object.keys`.

Reply:
Wait, what happens if `Object.keys(ghIssues).length === 0`? No filtering at all?

Reply:
If I remove this check there are a number of tests which fail :disappointed:

Reply:
Hmm, we should fix those failures then. Maybe the failing tests include some closed issues 😄

Reply:
Maybe. Fine then I'll update this in 10-15 mins